docs/pages/2019_IIT_Mandi_India/sections/first_taste.rst

```diff
@@ -52,23 +52,24 @@ Each piece of data in AiiDA gets a PK number (a "primary key") that identifies i
 The PK is printed to screen by the ``verdi data structure import`` command.
 **Mark down the PK for your structure and use it to replace the <PK> placeholders in what follows.**
 
-.. note::
+.. tip::
 
    You can view the structure either `online <http://crystallography.net/cod/9008565.html>`_
-   or use ``jmol 9008565.cif`` locally.
+   or by executing the ``jmol 9008565.cif`` command on the virtual machine.
 
 .. Let jason/jianxing test speed of SSH forwarding - potentially mention jupyter
 
 The following short python script sets up a self-consistent field calculation for the Quantum ESPRESSO code:
 
 .. literalinclude:: include/snippets/demo_calcjob.py
 
-Download the :download:`demo_calcjob.py <include/snippets/demo_calcjob.py>` script to your working directory.
+Download the :download:`demo_calcjob.py <include/snippets/demo_calcjob.py>` script using ``wget`` to your working directory.
```

Comment:
If they should use `wget`, it may be helpful that we provide the command or the link written out, I think?
Otherwise, I think the addition is superfluous as they can simply press the link or right-click and "Save As", I guess.

Reply:
They can't use "Save as" to directly store to the VM, can they? I'm ok with providing the command though.

Reply:
> They can't use "Save as" to directly store to the VM, can they?

I guess not, but they can `scp` over whatever they have locally to the VM anytime. But that will of course be yet another new thing, if they didn't know it already.

Reply:
Even assuming that 90% of the participants know how to use `wget` and `scp`, these hints are for the 10% who don't.

Reply:
Yeah, I guess you're right. With that in mind, a hint may be added about it, I suppose.

Reply:
We already introduced `wget` earlier.